proto/grpc.proto

```diff
@@ -36,19 +36,18 @@ message ConfirmItineraryRequest {
     string user_id = 2;
 }
 
-// Dummy message, will be replaced with
-// svc_storage_client_grpc::flight_plan::Object type
+// This is replaced by the FlightPlanObject from svc_storage
+//  during the build process. See build.rs
 message FlightPlanObject {
+    bool arbitrary = 1;
 }
 
 // Itinerary includes id, flight plan and potential deadhead flights
 message Itinerary {
     // itinerary id
-    string id = 1;
+    string itinerary_id = 1;
```

Comment:
Why are we deviating from the id field naming here?
All objects are defining an `id` field for their unique identifier, is this one different?

Reply:
https://github.com/Arrow-air/svc-scheduler/pull/39/commits/b98f89ac17fa9427aacb7e0021392d176fb42f10